packages/svelte/src/morfeoAction.ts

```diff
@@ -1,22 +1,32 @@
 import { getStyles } from '@morfeo/jss';
 import { component, Style } from '@morfeo/web';
 
-function getElementName({ componentName, variant }: Style) {
-  if (componentName && variant) {
-    return `${componentName}-${variant}`;
+function getElementName({ componentName, variant, state }: Style) {
+  let elementName = '';
+
+  if (componentName) {
+    elementName = `${componentName}`;
+  }
+
+  if (variant) {
+    elementName = `${elementName}-${variant}`;
+  }
+
+  if (state) {
+    elementName = `${elementName}-${state}`;
   }
 
-  return componentName || 'morfeo-element';
+  return elementName;
```

Comment:
Not every style specifies the properties `componentName` or `variant` or `state`, that's why in case the variable `elementName` was undefined the fallback was `morfeo-element`, how are you handling this now?